### ISSUE TITLE
uploaded files are moved to app folder with 0KB and are wiped from local storage

### ISSUE THREAD

**Issue by illBeRespawned** [nextcloud/android#2736]
### Actual behaviour

This issue happened on the Fairphone 2 (18.04.1)

I tried to manually upload pictures to nextcloud (auto upload was diabled and never used before on this device).

To my horror all pictures were moved to the app directory and were 0KB in size
They were also uploaded like that.

For whatever reason nextcloud not only deleted but pretty much wiped the pictures of the local storage source folder.

I rooted the phone and tried diskdigger which comes up with none of the missing pictures.

This is a terrible bug when your application to backup your pictures not only does not backup anything but wipes your files beyond repair.
This made me loose all of my holiday pictures and so far no luck in recovering them.

This seems to be somehow related to this issue:
https://github.com/nextcloud/android/issues/735
where i also commented

It is worth noting that the server might have been out of storage but still that should not wipe the pictures from local storage.

### Expected behaviour
- your pictures should stay on your phone and should be backed up by nextcloud without being destroyed beyond repair.
 
### Steps to reproduce
1. get fairphone 2
2. try to upload 300 pictures manually
3. try to find your pictures on the phone

### Environment data
Android version:

Device model: 
Fairphone 2 (18.04.1)
Stock or customized system:
stock
Nextcloud app version:
3.2.0
Nextcloud server version:
Nextcloud 10.0.1 (Business) hosted on ocloud.de

PHP

Version: 5.5.9

Memory Limit: 512.0 MB

Max Execution Time: 3600

Upload max size: 513.0 MB

Database

Type: mysql

Version: 5.5.52

Size: 64.7 MB

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by mario** [comment]
@AndyScherzinger as far as I know you can also select behaviour when doing manual upload? 

This is one of the reasons I want behaviours removed. They are tested on few phones and there are millions of phones out there, bunch of different SD cards and bunch of other factors that we can't take into consideration.

Either that, or somebody rewrites the whole thing and spends half a year testing this.

**Comment by AndyScherzinger** [comment]
> @AndyScherzinger as far as I know you can also select behaviour when doing manual upload?

Just checked and you are right @mario, behaviors are also available for manual upload.

**Comment by illBeRespawned** [comment]
guys i don't want to rain on your parade here but this is a application breaking bug.

not only did this backup app not backup any files it even destroyed them beyond repair.

I've spent hours trying to get them back. no luck.

Any help how to fix this desaster that the nextcloud app caused would be greatly apprciated.

I strongly vote for removing behaviours if that leads to loosing all your data if use the nextcloud app.
At this point the app behaves like an eraser tool not like a backup and sharing tool.

**Comment by mario** [comment]
cc @tobiasKaminsky

**Comment by illBeRespawned** [comment]
@mario 
it would be really helpfull to know what devices and OS versions this was tested on.
We roll out nextcloud for internal POCs right now and if it only works with one or two phones that would be very good to know. Is there a list or recommended phone/OS combination.

**Comment by mario** [comment]
@illBeRespawned there's nothing of the sorts atm. It SHOULD work on majority of phones and OSes, but like I said earlier - various vendors do various things with their phones and their Androids.

Were the pictures on internal storage or SD card btw?

**Comment by illBeRespawned** [comment]
@mario 
that's what i thought. 
this poses a pretty serious problem if you have no way to figure out if your phone will wipe out your files once you use manual upload. 

The pictures were stored on internal storage. 
I rooted the phone and tried diskdigger but not luck finding anything the anymore.

**Comment by dvn0** [comment]
I've now experienced this same bug via Firefox on my desktop.

**Comment by florianhof** [comment]
I also experienced 0kb files when exporting from the android app _TOTP Authentificator_ to my nextcloud listed on the drives. I tried on 2 nextcloud accounts on 2 providers with Nextcloud 22.2.3 in 2 different countries from my Sony phone on Android 11. I cannot see any relevant error message on the nextcloud server or in ne nextcloud android app. 

Generally speeking, the nextcloud android app is not reliable for uploading, and this over monthes. When uploading photos, there are very often problems with a few files, making it anoying to use, requiring several delete/retry rounds to achieve an upload. It's impossible for me to say if it's due to an android exporting app (photo, file manager, authenticator-export, etc.), to nextcloud android, to my nextcloud hoster, to nextcloud server or whatever. A control mechanism and (understandable) logs would be appreciated.
